sensing/autoware_pointcloud_preprocessor/schema/pickup_based_voxel_grid_downsample_filter_node.schema.json

```diff
@@ -1,46 +1,39 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "title": "Parameters for Pickup Based Voxel Grid Downsample Filter Node",
+  "title": "autoware_pointcloud_preprocessor parameters",
   "type": "object",
   "definitions": {
-    "pickup_based_voxel_grid_downsample_filter": {
+    "pickup_based_voxel_grid_downsample_filter_node": {
       "type": "object",
       "properties": {
         "voxel_size_x": {
           "type": "number",
-          "description": "voxel size along the x-axis [m]",
-          "default": "1.0",
-          "exclusiveMinimum": 0.0
+          "description": "Voxel size along the x-axis [m].",
+          "default": 1.0,
+          "minimum": 0.0
         },
         "voxel_size_y": {
           "type": "number",
-          "description": "voxel size along the y-axis [m]",
-          "default": "1.0",
-          "exclusiveMinimum": 0.0
+          "description": "Voxel size along the y-axis [m].",
+          "default": 1.0,
+          "minimum": 0.0
         },
         "voxel_size_z": {
           "type": "number",
-          "description": "voxel size along the z-axis [m]",
-          "default": "1.0",
-          "exclusiveMinimum": 0.0
+          "description": "Voxel size along the z-axis [m].",
+          "default": 1.0,
+          "minimum": 0.0
         }
       },
       "required": ["voxel_size_x", "voxel_size_y", "voxel_size_z"],
       "additionalProperties": false
     }
   },
   "properties": {
-    "/**": {
-      "type": "object",
-      "properties": {
-        "ros__parameters": {
-          "$ref": "#/definitions/pickup_based_voxel_grid_downsample_filter"
-        }
-      },
-      "required": ["ros__parameters"],
-      "additionalProperties": false
+    "ros__parameters": {
+      "$ref": "#/definitions/pickup_based_voxel_grid_downsample_filter_node"
     }
   },
-  "required": ["/**"],
+  "required": ["ros__parameters"],
```

Comment:
This should stay the same as before, otherwise it will not be read correctly by ROS or the markdown generator.